### PR TITLE
Add velocity control interfaces

### PR DESCRIPTION
**Pull request by stephanie-eng**
Add velocity control interfaces to the driver and IRB1200 5/0.9
Update RAPID example code as per:
https://github.com/ros-industrial/abb_libegm/issues/64

### CODE REVIEW

**Comment by AndyZe** [comment]
Can you add some testing instructions to the PR? How did you test it? I'm curious if you tried sending the 3 combos: position commands only, vel commands only, and pos+vel commands. It should work either way

File: abb_bringup/launch/abb_control.launch.py [hunk 1]
```diff
@@ -145,10 +145,7 @@ def generate_launch_description():
         package="controller_manager",
         executable="ros2_control_node",
         parameters=[robot_description, robot_controllers],
-        output={
-            "stdout": "screen",
-            "stderr": "screen",
-        },
+        output="both",
```
Comment (AndyZe) [inline abb_bringup/launch/abb_control.launch.py:160]: Do you have a link for this -- either documentation or somebody else using it? I haven't seen it before

**Review by AndyZe — APPROVED** [review]
Approving assume a link is provided for `output="both"`  :+1:

File: abb_bringup/launch/abb_control.launch.py (hunk 1 above)
Comment (stephanie-eng) [reply abb_bringup/launch/abb_control.launch.py:160]: Henning told me about this here:
https://github.com/ros-planning/moveit_resources/pull/120#discussion_r839768676

We recently changed this in moveit_resources here:
https://github.com/ros-planning/moveit_resources/pull/129

**Pull request merged**